src/pods/movies/actions/index.ts

```diff
@@ -0,0 +1,14 @@
+import { MovieFilter } from "../viewModel";
+import { BaseAction } from "../../../common-app";
+import { actionsEnums } from "../../../common-app/actions";
+import { MovieEntity } from "../components/view-model";
+
+export const moviesRequestAction = (filter?:MovieFilter): BaseAction<MovieFilter> => ({
+	type: actionsEnums.MOVIES_REQUEST,
```

Comment:
I would name this action: 

MOVIES_REQUEST_START

Just to be consistent with:

MOVIES_REQUEST_START
MOVIES_REQUEST_COMPLETED